package.json

```diff
@@ -36,6 +36,7 @@
     "gulp": "^3.8.10",
     "istanbul": "^0.4.2",
     "mocha": "^2.0.1",
+    "multipipe": "^0.3.0",
```

Comment:
@j3parker I think @Surekh is correct, `multipipe` should be a normal dependency.